algorithms/Lesson2_Arrays/ex_1.rb

```diff
@@ -0,0 +1,8 @@
+def solution(array)
+  counter = Hash.new 0
+  array.each.with_index do |num, _i|
+    counter[num] += 1
+  end
+  odd_values_array = counter.values.select(&:odd?)
```

Comment:
when you're using only first element from selected array, please use `find` instead of `select` https://ruby-doc.org/core-2.2.3/Enumerable.html#method-i-find

(find will return first matching element instead of always passing through the whole array)